components/script/dom/globalscope.rs

```diff
@@ -94,7 +95,7 @@ impl Drop for AutoCloseWorker {
 pub struct GlobalScope {
     eventtarget: EventTarget,
     crypto: MutNullableDom<Crypto>,
-    next_worker_id: Cell<WorkerId>,
+    next_worker_id: WorkerId,
```

Comment:
I don't think that field is necessary anymore. Just create a new uuid when necessary.